docs/open-source-projects.mdx

```diff
@@ -412,6 +412,20 @@ export const projectsData = [
 
 ### Local Development Environments
 
+EOS Costa Rica offers a wide variety of local development environments for the developer community.
+
+<div class="table-wrapper" markdown="block" style={{overflow: 'scroll', height: '200px'}}>
+
+| Local Development Environments | Description | 
+| :----------: | :---------: | 
+| [EOS](https://github.com/eoscostarica/eos-local) | Docker-based EOS Local Network for Development. | 
+| [Telos](https://github.com/eoscostarica/telos-local)| Docker-based Telos Network for Development. |
+| [Proton](https://github.com/eoscostarica/proton-local)| Docker-based Proton Network for Development. | 
+| [Wax](https://github.com/eoscostarica/wax-local) | WAX Local Development Environment Docker Image. | 
+| [EOSIO Explorer](https://github.com/eoscostarica/eosio-explorer)| An application providing Web GUI to communicate with EOSIO blockchain in a local development environment. | 
```

Comment:
If you agree with [my comment in the Spanish version](https://github.com/eoscostarica/guias.eoscostarica.io/pull/292/files/6cf1dc4aceba49483e7f8949263a25cd9411a02f..9dd0bc7f7eb7067dfa88ea0ab0b0546b93ac291c#r787405363), make sure to update it here too